tensorrt_llm/_torch/model_config.py

```
@@ -336,19 +337,15 @@ def get_bindings_model_config(self,
-                architectures = self.pretrained_config.architectures
-                if len(architectures
-                       ) == 1 and architectures[0] == "DeciLMForCausalLM":
-                    mlp_hidden_size = self._infer_nemotron_ffn_mult()
-                else:
-                    raise ValueError(
-                        f"Inferring mlp hidden size for model architecture: {architectures} isn't supported yet"
-                    )
+            mlp_hidden_size = self._infer_nemotron_ffn_mult()
+        else:
+            raise ValueError(
+                f"Inferring mlp hidden size for model architecture: {self.pretrained_config.architectures} isn't supported yet"
```

Ruff (0.12.2)

Line too long (127 > 120)

(E501)